### PR TITLE
Requests: Add Outputs requests

### PR DESCRIPTION
**Pull request by Brendonovich**
### Description

Adds the following requests as documented in the planned requests sheet:
- GetOutputList
- GetOutputStatus
- ToggleOutput
- StartOutput
- StopOutput
- GetOutputSettings
- SetOutputSettings

### Motivation and Context

Combined with #929, this should be the last of the requests that need to be implemented for v5 to have feature parity with v4, thus making it ready for release.

### How Has This Been Tested?

Tested OS(s): 
All requests were tested with OBS 27.2.2 on macOS 11.6.1

### Types of changes

- New request/event (non-breaking)

### Checklist:

-  [x] I have read the [Contributing Guidelines](https://github.com/obsproject/obs-websocket/wiki/Contributing-Guidelines).
-  [x] All commit messages are properly formatted and commits squashed where appropriate.
-  [x] My code is not on the `master` branch.
-  [x] The code has been tested.
-  [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by DeeJayMX** [comment]
👍👍👍👍👍

**Comment by Brendonovich** [comment]
@DeeJayMX Please stop leaving these comments, they aren't helpful. If you find an issue with the PR then please let me know, but otherwise I am only looking for approval from the project's maintainers.

**Comment by DeeJayMX** [comment]
Ok I just want this function from the beginning of the OBS Project. I just wanted to encourage it.
But ok I will not 🥲

File: src/requesthandler/RequestHandler_Outputs.cpp [hunk 1]
```diff
@@ -275,3 +275,217 @@ RequestResult RequestHandler::GetLastReplayBufferReplay(const Request&)
...
+{
+	json responseData;
+	responseData["outputs"] = Utils::Obs::ArrayHelper::GetOutputList();
+	return RequestResult::Success(responseData);
+}
+
+/**
+ * Gets the status of an output.
+ *
+ * @requestFied outputName | String | Output name
```
Comment (ruggi99) [inline src/requesthandler/RequestHandler_Outputs.cpp:299]:
```suggestion
 * @requestField outputName | String | Output name
```
Is this a typo?

Comment (Brendonovich) [reply src/requesthandler/RequestHandler_Outputs.cpp:299]: Yes, thanks for pointing that out!

**Pull request merged**